### PR TITLE
Reduce coupling edit

### PR DESCRIPTION
**Pull request by ntache-81**
Transferred comments/edits from Gdoc to md.

### CODE REVIEW

File: practices/reduce-coupling-between-abstractions.md [hunk 1]
```diff
@@ -23,46 +28,49 @@ Interfaces should be used judiciously, primarily where they provide flexibility
...
-Start with the most problematic areas, gradually refactoring and decoupling components, which helps manage risk and maintain system stability.
+Start with the most problematic areas, gradually refactoring and decoupling components. This helps manage risk and maintain system stability.
 
 ### Recognizing Natural Coupling
 
 Not all coupling is bad; some level of dependency is natural and necessary.
 Recognizing and accepting necessary coupling helps avoid futile efforts to decouple what should inherently be connected.
 
 ### Design for Change vs. YAGNI (You Aren't Gonna Need It)
+<ed> I'm not sure I understand the specific pitfall/challenge from this title. Perhaps "Design for Needed, Not Speculative, Features"? </ed>
```
Comment (dcmoore) [inline practices/reduce-coupling-between-abstractions.md:42]: Correct. The more assumptions a developer makes about future changes that may not actually come to bear, the more complexity they may needlessly introduce.

File: practices/reduce-coupling-between-abstractions.md [hunk 2]
```diff
@@ -23,46 +28,49 @@ Interfaces should be used judiciously, primarily where they provide flexibility
...
-- **Refactor**: Set some time aside to refactor a key component or set of components to reduce coupling. Present your findings to the team to see if committing those changes or making additional changes have a good potential return on investment.
-- **Audit Your Dependencies**: Use a dependency analysis tool to visualize the relationships between modules and components, identify highly coupled areas, and discuss why these dependencies exist.
+### [Start a Book Club](/practices/start-a-book-club.md)
 
-## Supporting Capabilities
+#### [Clean Architecture by Robert C. Martin](https://www.goodreads.com/book/show/18043011-clean-architecture) 
+This book delves into principles and practices that ensure code remains clean, emphasizing the importance of separation of concerns and the decoupling of systems for better manageability.
 
-### [Code Maintainability](/capabilities/code-maintainability.md)
+<ed> Are there any text/video resources on refactoring, specifically, that you want to recommend? </ed>
```
Comment (dcmoore) [inline practices/reduce-coupling-between-abstractions.md:66]: Yeah, let's add two more:
1) [Working Effectively with Legacy Code](https://www.goodreads.com/book/show/44919.Working_Effectively_with_Legacy_Code) - Discusses how to find seams, add automated test coverage, and refactor the system to be more simple.
2) [Refactoring](https://www.goodreads.com/en/book/show/44936.Refactoring) - Similar to the above book, but covers the content from a first principles standpoint.

File: practices/reduce-coupling-between-abstractions.md [hunk 3]
```diff
@@ -86,3 +94,24 @@ Clear abstractions and minimal dependencies support better documentation and col
 * Have we identified the most problematic areas of coupling in our codebase?
 * What small, incremental changes can we make to start reducing coupling in these areas?
 * How do we ensure system stability while refactoring to reduce coupling?
+
+
+### [Do a Spike](/practices/do-a-spike.md)
+
+<ed> This may be a silly question, but does your target reader know what a spike is? I'm assuming they do and we don't need to define this practice, but just want to double-check. </ed>
```
Comment (dcmoore) [inline practices/reduce-coupling-between-abstractions.md:101]: I would guess they do. We can update the name to something like, "timebox an experiment" or something.

File: practices/reduce-coupling-between-abstractions.md [hunk 4]
```diff
@@ -86,3 +94,24 @@ Clear abstractions and minimal dependencies support better documentation and col
...
+- **Audit Your Dependencies**: Use a dependency analysis tool to visualize the relationships between modules and components, and to identify highly coupled areas. Discuss why these dependencies exist. <ed> Is there a specific tool(s) you'd recommend? </ed>
+
+
+## Supporting Capability
+
+### [Code Maintainability](/capabilities/code-maintainability.md)
+
+Reducing coupling between abstractions enhances the Code Maintainability capability by creating a modular and flexible codebase.
+Independent, well-defined components minimize unintended side effects, making the code easier to understand, modify, and test.
+This modularity ensures that changes in one part of the system do not disrupt others, preserving stability and reducing cognitive load. <ed> Cognitive load on the developers or on the system? </ed>
```
Comment (dcmoore) [inline practices/reduce-coupling-between-abstractions.md:114]: On the mind of the developers who maintain the system.

File: practices/reduce-coupling-between-abstractions.md (hunk 1 above)
Comment (ntache-81) [reply practices/reduce-coupling-between-abstractions.md:42]: Got it. Will make this change to the header title.

File: practices/reduce-coupling-between-abstractions.md (hunk 2 above)
Comment (ntache-81) [reply practices/reduce-coupling-between-abstractions.md:66]: Will do.

File: practices/reduce-coupling-between-abstractions.md (hunk 3 above)
Comment (ntache-81) [reply practices/reduce-coupling-between-abstractions.md:101]: Ok, I will add this clarification to the running text.

File: practices/reduce-coupling-between-abstractions.md (hunk 4 above)
Comment (ntache-81) [reply practices/reduce-coupling-between-abstractions.md:114]: Got it. Will make this edit in the file.

File: practices/reduce-coupling-between-abstractions.md [hunk 5]
```diff
@@ -86,3 +94,24 @@ Clear abstractions and minimal dependencies support better documentation and col
...
 * How do we ensure system stability while refactoring to reduce coupling?
+
+
+### [Do a Spike](/practices/do-a-spike.md)
+
+<ed> This may be a silly question, but does your target reader know what a spike is? I'm assuming they do and we don't need to define this practice, but just want to double-check. </ed>
+
+
+- **Refactor**: Set some time aside to refactor a key component or set of components to reduce coupling. Present your findings to the team to see if committing those changes or making additional changes have a good potential return on investment.
+- **Audit Your Dependencies**: Use a dependency analysis tool to visualize the relationships between modules and components, and to identify highly coupled areas. Discuss why these dependencies exist. <ed> Is there a specific tool(s) you'd recommend? </ed>
```
Comment (ntache-81) [inline practices/reduce-coupling-between-abstractions.md:105]: Is there a specific tool(s) you'd recommend?

Comment (dcmoore) [reply practices/reduce-coupling-between-abstractions.md]: Sorry, I didn't see this until just now. You can link to this guy's website who does a decent job at listing what's out there.

Comment (dcmoore) [reply practices/reduce-coupling-between-abstractions.md]: https://markgacoka.medium.com/how-to-visualize-your-codebase-7c4c4d948141

Comment (ntache-81) [reply practices/reduce-coupling-between-abstractions.md]: Done!

**Pull request merged**